src/App.jsx

```diff
@@ -1,13 +1,30 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { Route, Routes } from "react-router-dom";
+
+import "./App.css";
+import Home from "./Home";
+import Empty from "./shared/empty";
+import Login from "./shared/login";
 
 const queryClient = new QueryClient();
 
 function App() {
   return (
     <QueryClientProvider client={queryClient}>
-      <h1 className="text-3xl font-bold underline text-mainSkyblue">
-        Hello world!
-      </h1>
+      <Routes>
+        <Route
+          path="/"
+          element={<Home />}
+        />
+        <Route
+          path="/Empty"
```

Comment:
p4: 저장된 history가 없는 페이지이지만, 저희 서비스를 이용하는 사용자에게는 처음 보여지는 페이지이므로 이를 활용하여 네이밍을 하면 좋을 것 같습니다.
```suggestion
          path="/initial"
```

Reply:
의견주셔서 감사합니다:)